container/deploy_openqa_containers.sh

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
+set -e
+
+FORCE=0
+PREFIX="openqa"
+NETWORK="${PREFIX}_net"
+PORT="80"
+ACTION=""
+WEBUI_IMAGE="registry.opensuse.org/devel/openqa/containers15.2/openqa_webui:latest"
+WORKER_IMAGE="registry.opensuse.org/devel/openqa/containers15.2/openqa_worker:latest"
+
+function wait_for_container_ready {
+  container=$1
+
+  for ((count=0; count<30; count++))
+  do
+    sleep 1
+    if docker inspect openqa2_db | grep '"Status": "running"' >/dev/null; then
+      echo "OK - Container $container started correcty"
+      break
+    fi
+  done
+}
+
+function wait_for_container_appropiate_logs {
+  container=$1
+  message=$2
+
+  if [ -n "$3" ]; then
+    countMax=$(($3))
+  else  
+    countMax=30
+  fi
+
+  for ((count=0; count<countMax; count++))
+  do
+    if docker logs "$container" 2>&1 | grep "$message" >/dev/null; then
+      echo "OK - Container $container stand-up correctly - found in logs $message"
+      break
+    fi    
+  done
+
+  if [[ $count -eq countMax ]]; then
+    echo "ERROR - Sorry the container $container didn't stand-up correctly"
+    exit 2
+  fi
+}
+
+function show_usage {
```

Comment:
The usage is detailed but leaves some important questions: What *does* this script and what are the pre-conditions to run it? Do I need to run it as root? Is it supposed to run within a container? What is meant with "deployment" (which is a rather generic term)? Where is it deployed to? Maybe the script's name could also be changed to be more specific.